jstp/src/main/java/com/metarhia/jstp/JSTP.java

```diff
@@ -7,17 +7,32 @@
 
 /**
  * Common JSTP interface
+ *
  * Created by lidaamber on 18.06.16.
  */
 public final class JSTP implements Serializable {
 
   private JSTP() {
   }
 
+  /**
+   * Parses string representation of JS value
+   *
+   * @param input string representation of JS value
+   *
+   * @return parsed JS value
```

Comment:
Also, we should put that description to the JSParser.parse and just have a link here.